faiss/IndexPQ.h

```diff
@@ -26,10 +26,6 @@ struct IndexPQ : IndexFlatCodes {
     ProductQuantizer pq;
 
     /** Constructor.
```

Comment:
if there is a choice, please priviledge putting the doc in a /** */ in front of the function rather than inline (inline is worse with multi-line doc)